lib/js/dom/structure.js

```diff
@@ -0,0 +1,118 @@
+// The controls above the input panel
+const btnReset = {
+  tagName: 'button',
+  className: 'btn-alt btn-reset js-btn-reset',
+  textContent: 'Reset'
+};
+
+const btnShortUrl = {
+  tagName: 'button',
+  className: 'btn-alt btn-short-url js-btn-short-url',
+  textContent: 'Make Short URL:'
+};
+
+const urlOut = {
+  tagName: 'input',
+  className: 'url-out js-url-out',
+  attrs: {
+    type: 'text'
+  }
+};
+
+// The input panel
+const inputPanel = {
```

Comment:
Oh, I didn't know about `rdom`. Looks good! If you don't mind I'll put this PR on hold and will update it with `rdom` (should be this week).